tests/mpitestladds/distributedMemParallelization/RegularGridDecompositionTests.cpp

```diff
@@ -0,0 +1,273 @@
+/**
+ * @file RegularGridDecompositionTests.cpp
```

Comment:
seems to have some overlap with AltitudeBasedDecompositionTests.cpp. Could probably benefit from some common super class for setup stuff.